lib/utils/utils.dart

```diff
@@ -26,6 +26,10 @@ class StorageKeys {
   static const codeThemeDark = 'code-theme-dark';
   static const iCodeFontSize = 'code-font-size';
   static const codeFontFamily = 'code-font-family';
+  static const defaultStartTabGh = 'default-start-tab-github';
+  static const defaultStartTabGl = 'default-start-tab-gitlab';
+  static const defaultStartTabBb = 'default-start-tab-bitbucket';
+  static const defaultStartTabGt = 'default-start-tab-gitea';
```

Comment:
If the start tab is set to another, how would the other tabs be arranged?

Reply:
I only changed the default active tab. I haven't changed the arrangement of the tabs.

Reply:
Oh, get it. So if "notification" is set to the default tab, the order of tabs has no change, but the active tab would be the second on the first launch. Is that correct?

Reply:
Yes that's correct.

Reply:
OK. Just want to ask the issue author if it is as expected, while seems the account was deleted 😂

Reply:
Yeah I saw that. 😂Anyway, this is better than changing the order right? There's consistency.

Reply:
To be honest I'm not sure which way is better. In my opinion, seems there are few apps in which the active tab is not the first...

I'm not an expert in UI/UX. Is there any example app that has a behavior like this?

Reply:
BTW if you are working on other issues, you could checkout a new branch from master.

I'll help to solve the conflicts when this PR gets merged.

Reply:
In Play Store the default active tab is not the first. I just checked. They don't have a settings menu to change the default but they change it based on your app usage.